src/renderer/src/hooks/useNeonBackup.ts

```diff
@@ -112,7 +112,9 @@ const backupMigrationsNeo3Schema = zod.record(
     hash: zod.string(),
     neoLegacyAccount: backupAccountSchema,
     neo3Address: zod.string(),
-    status: zod.union([zod.literal('failure'), zod.literal('pending'), zod.literal('done')]),
+    status: zod
+      .union([zod.literal('check-failure'), zod.literal('failure'), zod.literal('pending'), zod.literal('done')])
+      .transform(value => (value === 'failure' ? 'check-failure' : value)),
```

Comment:
Remove the `failure` and execute a parser in migration from `failure` to `check-failure`.

Reply:
I turned the type back to "failure"